tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -237,10 +237,14 @@ export class ScalarCardComponent<Downloader> {
           datum.points[findClosestIndex(datum.points, startStep)];
         const selectedStepData: SelectedStepRunData = {};
         selectedStepData.COLOR = metadata.color;
+        let alias = '';
```

Comment:
alias is only used by ColumnHeaders.RUN case, so seems like you could move this code (L240 to L243) into that case block. Keep the variable closer to where it is used.